### ISSUE TITLE
Accessibility Bug: Narrator announces "Exiting Table" while the focus remains in the table and after that rows of the table are being read out

### ISSUE THREAD

**Issue by heatherbshapiro** [tabalinas/jsgrid#1214]
1. Bug Reports:
    A) Expected Behavior

When a user uses a screen reader, like Narrator, Narrator should not announce "Exiting Table" while the focus remains in the table and rows of the table are being read out. 

    B) Actual Behavior

Because jsgrid implements the grid as two tables, one for header and one for content, Narrator announces "Exiting Table" while the focus remains in the table and after that rows of the table are being read out.
    C) Steps to Reproduce
When a user uses “caps lock” + right arrow to access from the last element of header row to the first element of first row The narrator says “table exits” 

User Impact
End User will be get the impression that the table control does not contain all the other items i.e. when the narrator further reads the rows of the table.

**Comment by mgifford** [comment]
Would love to know if there has been any progress with this. Also, has any other accessibility work been done on jsgrid?
